en/deploy-tidb-from-kubernetes-gke.md

```diff
@@ -98,25 +98,17 @@ If you see `Ready` for all nodes, congratulations! You've set up your first Kube
 TiDB Operator uses [Custom Resource Definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) to extend Kubernetes. Therefore, to use TiDB Operator, you must first create the `TidbCluster` CRD.
 
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
+kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.2/manifests/crd.yaml && \
```

Comment:
install the same version as the tidb-operator

Reply:
I don't like this very much, because it means we have to remember to update this every time we have a new release.

Reply:
yes, we should update this but the tutorial still work even if we forget